src/card.css

```diff
@@ -0,0 +1,87 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 1.4;
+  color: var(--color-main-text);
+}
+
+.card {
+  margin: 50px auto;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 400px;
+  border: 1px solid var(--main-card-border-color);
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: var(--section-card-font-size_medium);
+  line-height: 1.5;
+  margin-bottom: 4px;
+  color: var(--color-dark-text);
+}
+
+.card__product-code {
+  line-height: 1.4;
+  margin-bottom: var(--section-card-bottom-gap);
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__reviews-count {
+  color: var(--color-dark-text);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: var(--section-card-bottom-gap);
+}
+
+.card__price-name {
+  font-weight: 400;
+  font-size: var(--section-card-font-size_medium);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-dark-text);
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  background: var(--section-card-color-button-blue);
+  color: var(--section-card-color-button-white);
+  transition: 0.5s;
```

Comment:
Add a transition property too. Now it applies to all the properties, which is not good for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x